buildSrc/src/main/java/org/elasticsearch/gradle/test/RestApiSpecExtension.java

```diff
@@ -0,0 +1,70 @@
+package org.elasticsearch.gradle.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RestApiSpecExtension {
+    private List<String> includesCoreSpec = new ArrayList<>();
+    private List<String> includesCoreTests = new ArrayList<>();
+    private List<String> includesXpackSpec = new ArrayList<>();
+    private List<String> includesXpackTests = new ArrayList<>();
+    private boolean alwaysCopySpec = false;
+    private boolean copyTests = false;
+    private boolean copyXpackTests = false;
+
+    public void includeCoreSpec(String include) {
```

Comment:
tip: this could be `String... include` to make the DSL a little less verbose allowing for things like `includeCoreSpec 'foo', 'bar'`.